src/defs.h

```diff
@@ -896,7 +896,7 @@ str_strip_prefix_len(const char *str, const char *prefix, size_t prefix_len)
  * part.
  */
 # define QUOTE_HEXSTR_MASK			(0x3 << QUOTE_HEXSTR_SHIFT)
-static_assert((NUM_HEXSTR_OPTS - 1) <= (QUOTE_HEXSTR_MASK >> QUOTE_HEXSTR_SHIFT),
+_Static_assert((NUM_HEXSTR_OPTS - 1) <= (QUOTE_HEXSTR_MASK >> QUOTE_HEXSTR_SHIFT),
```

Comment:
Why this change is needed?

Reply:
We have static_assert.h (included by defs.h) to handle this on various platforms.
Please have a look why it didn't work out in your case.